styles/config/vocabularies/Docs/accept.txt

```diff
@@ -69,7 +69,7 @@ CocoaPods
 Codecov
 CodeDeploy
 Coinbase
-[Cc]onfig
+[Cc]onfig\b
```

Comment:
Is this a typo?

Reply:
No you can ignore this it's just the linter settings